template.json

```diff
@@ -288,31 +358,34 @@
       "annihilation" : ""
     }
   },
-  "Item": {
-    "types": [
+  "Item" : {
+    "types" : [
       "ammunition",
       "armour",
       "archetype",
       "augmetics",
       "equipment",
       "injury",
+      "misc",
```

Comment:
Could "Equipment" cover miscellaneous?

Reply:
Is it possible for Equipment to have a mix of null Value/Rarity/Keywords/Effect/Quantity?  If so, I would think that should be fine!

Example of misc:
Trinket - Contains Description, Quantity.  Nothing else.
Quest Item - Contains Description, Effect?.

Reply:
The possible values are anything I want it to be, so if I want to enforce it to not be null, I can do so, or I can allow it. So yes, they can certainly have null.